components/Navbar.vue

```diff
@@ -39,7 +39,7 @@
         alt="GitHub Icon"
         height="15px"
         width="15px"
-      />
+      >
```

Comment:
Cant it be formatted in a better way, having just a tag closer in line doesn't look good, Was it automatic lint fix or something else?